scripts/pre-start/kubernetes_pre_start_template.py

```diff
@@ -0,0 +1,55 @@
+import argparse
+import json
+from typing import Dict
+
+from kubernetes import client, config
+
+{{ config_code }}
```

Comment:
Have you tried testing with an invalid script? What does the error look like? Is it hard to debug?

Reply:
you should be able to see errors in the init container logs if it fails. we could potentially show those error logs in the UI